src/state/queries/actor-autocomplete.ts

```diff
@@ -20,6 +20,7 @@ export const RQKEY = (prefix: string) => [RQKEY_ROOT, prefix]
 export function useActorAutocompleteQuery(
   prefix: string,
   maintainData?: boolean,
+  limit?: number,
```

Comment:
Really shouldn't use positional props but didn't want to make edits in all the other files

Reply:
yea all good, felt the same way when i added the `maintainData`